test/passing/tests/sequence-preserve.ml.ref

```diff
@@ -75,9 +75,8 @@ let foo x y =
 
 let foo x y =
   do_some_setup x ;
-  let () = do_some_setup y in
+  let () = do_some_setup y in (* Empty line after let *)
```

Comment:
This formats like the input, which is nice. However, it's also the output of:
```ocaml
  let () = do_some_setup y in
  (* Empty line after let *)
```

I think it's better to break the line before the comment to make the `in` clearer. (`Cmt.fmt_after` has an optional argument or this)
Also, this case needs some care:

```ocaml
  important_function x ;
  let () = do_some_setup y in
  (* Empty line after let *)
```

Reply:
I agree your proposal is better, however having looked around a bit, I think `fmt_after` is not directly used here, and because of this I am not sure how to proceed.